### ISSUE TITLE
[Android] Display favorite tracks in Recently Scrobbled list, and add ability to like/unlike tracks ❤️

### ISSUE THREAD

**Issue by Harmonie666** [nrubin29/finale#14]
It would be nice if liked songs would be displayed in finale, and if we were able to like/unlike songs in finale, like PanoScrobbler let's us to do so.

![Screenshot_20220222-001712948.png](https://user-images.githubusercontent.com/73917917/155037365-f08c6f52-fe62-41a5-9980-8370a18909ad.png)

![Screenshot_20220222-001712948 (1).png](https://user-images.githubusercontent.com/73917917/155037373-af1049ac-0070-4ac0-8708-09e62d3bd874.png)

**Comment by nrubin29** [comment]
You can like/unlike a song and check its status on the track page, but I could look into add that functionality to the profile page.

**Comment by Harmonie666** [comment]
> You can like/unlike a song and check its status on the track page

Yes I realized that shortly after typing this post. It would still be nice if this was possible on the front page like PS. By adding the 3 dots to now playing track and recently scrobbled ones.

**Comment by nrubin29** [comment]
Showing a heart icon on loved songs in the recent scrobbles list was implemented in https://github.com/nrubin29/finale/commit/a13e4d2cff5b33324573f623b085f7ba55388dda. I'll look into adding the ability to love songs from the recent scrobbles list in the future.
